R/google_trends.R

```diff
@@ -106,24 +105,8 @@ getGoogleTrends <- function(user, password, query = "", type = "trend", last = "
     tidyr::gather_(bind_data, "keyword", "trend", colnames(bind_data)[2:ncol(bind_data)], na.rm = TRUE)
   } else if (type == "trend"){
     trend <- ret[[type]]
-    # use query and geo arguments as column names to prevent garbled characters
-    # The query is lowered and spaces are changed into . by gtrendsR, so doing the same
-    queries <- stringr::str_to_lower(stringr::str_replace(stringr::str_trim(query), " +", "."))
-    cols <- if(!is.null(geo)){
-      # put geo into column names and separate it later
-      # spaces in queries are converted to . by gtrends, so space can be used safely
-      paste(queries, geo, sep = " ")
-    } else {
-      queries
-    }
-    colnames(trend)[(ncol(trend)-length(cols)+1):ncol(trend)] <- cols
-    # gather columns except for time columns to make it easy to visualise
-    ret <- tidyr::gather_(trend, "keyword", "trend", cols, na.rm = TRUE)
-    if(!is.null(geo)){
-      # separates geo and keywords that were column names
-      ret <- tidyr::separate_(ret, "keyword", c("keyword", "geo"), sep = " ")
-    }
-    ret
+    colnames(trend)[colnames(trend) == "hits"] <- "trend"
```

Comment:
If gtrends itself is changing that much, I think we don't need to try too hard to keep backward compatibility of out output. So I think returning "hits" columns as it is is fine.